lib/node_modules/@stdlib/utils/is-uppercase/test/test.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isUppercase = require( './../lib' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.equal( typeof isUppercase, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function returns `true` if provided an uppercase string', function test( t ) {
+	var values;
+	var bool;
+	var i;
+
+	values = [
+		'HELLO WORLD!',
+		'JUST WAIT, SOON YOU WILL REST AS WELL.'
+	];
+
+	for ( i = 0; i < values.length; i++ ) {
+		bool = isUppercase( values[ i ] );
+		t.equal( bool, true, 'returns true when provided '+values[ i ] );
+	}
+	t.end();
+});
+
+tape( 'the function returns `false` if not provided an uppercase string', function test( t ) {
+	var values;
+	var i;
+
+	values = [
+		'abc',
+		'Hello World',
+		void 0,
```

Comment:
Can we add an empty string, a string containing a punctuation character, and another containing only numbers?